pkgs/development/compilers/hashlink/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib, stdenv, fetchFromGitHub, libpng, libjpeg_turbo, libvorbis, SDL2, mesa_glu, mbedtls, openal, libuv }:
+
+stdenv.mkDerivation rec {
+  pname = "hashlink";
+  version = "1.11";
+
+  src = fetchFromGitHub {
+    owner = "HaxeFoundation";
+    repo = "hashlink";
+    rev = version;
+    sha256 = "sha256-Mw0AMPK4fdaAdq+BjnFDpo0B9qhTrecD8roLA/JF/a0=";
+  };
+
+  buildInputs =
+    [ libpng libjpeg_turbo libvorbis SDL2 mesa_glu mbedtls openal libuv ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    mkdir -p $out/lib
+    mkdir -p $out/include
+    cp hl $out/bin
+    cp libhl.so *.hdll $out/lib
+    cp src/hl.h src/hlc.h src/hlc_main.c $out/include
+  '';
+
+  dontPatchELF = true;
+  dontStrip = true;
```

Comment:
Every binary should be stripped. Why don't we want to patchelf?

Reply:
I don't know what this is for. I just copied this from compilers/neko which is the most similar thing I can think of.

Reply:
I've removed this